system/security/shared/seclib.hpp

```diff
@@ -55,9 +55,9 @@ enum SecAccessFlags : int
     SecAccess_Full = 255
 };
 
-static const char * getSecAccessFlagName(SecAccessFlags flag)
+inline const char * getSecAccessFlagName(SecAccessFlags flag)
 {
-	switch (flag)
+    switch (flag)
```

Comment:
indent: should you change the indentation of the rest of the function to remove the tabs?